Toggl.Core.UI/ViewModels/Reports/ReportsViewModel.cs

```
@@ -151,10 +155,14 @@ private IObservable<ImmutableList<IReportElement>> reportElements(ReportFilter f
             .ToObservable()
             .StartWith(createLoadingStateReportElements());
 
-        private string formattedTimeRange(DateTimeOffsetRange range, DateFormat dateFormat)
+        private string formattedTimeRange(DateRange range, DateFormat dateFormat)
         {
-            var startDateText = range.Minimum.ToString(dateFormat.Short, DateFormatCultureInfo.CurrentCulture);
-            var endDateText = range.Maximum.ToString(dateFormat.Short, DateFormatCultureInfo.CurrentCulture);
+            var knownShortcut = calendarShortcutsService[range];
+            if (knownShortcut != null)
```

I actually don't like that you return `null` if you don't find anything. What if we used a `TryGet(range, out var knownShortcut)`? I would prefer that over the indexer. What do you think about that?

---

You may not like it but it's precisely what `null` is for: the lack of value.
I don't want to do `out` vars for that reason. 
I will, however change the indexer.

---

Now that it's not an indexer, I don't mind the `null` return value (I believe the indexer should throw "out of bounds" instead of returning `null`).